ide/mplabx/include.am

```diff
@@ -0,0 +1,8 @@
+# vim:ft=automake
+# All paths should be given relative to the root
+
+EXTRA_DIST+= ide/mplabx/README.md
```

Comment:
Is it okay that some of the files are missing here?

Reply:
I was hesitant because of how large the auto generated MPLABX files are. Adding them though makes sense. I'll update it.